cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-user-uuids.cy.js

```diff
@@ -137,5 +137,24 @@ describe('bulk-edit', () => {
       BulkEditActions.openInAppStartBulkEditFrom();
       BulkEditActions.verifyCalendarItem();
     });
+
+    it('C359585 Verify clicking on the "Commit changes" button (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
```

Comment:
It looks like that there is no need in permission: "Users: Can view user profile". So we can remove this permission from the test case

Reply:
done